gfam/tasks/find_unassigned/task.py

```diff
@@ -27,102 +28,75 @@ def __init__(self,
         else:
             self.log = logger
         self.sequence_id_regexp = sequence_id_regexp
+        self.assignment_reader = None
+        self.seg_reader = None
 
         # these are moved here from get_unassigned
         self.regions = []
-        self.maximum = None
+        self.maximum = max(self.min_length,
+                           self.min_fragment_length)
+
         # to move somewhere else
         self.seqcat = {}
```

Comment:
This variable is now unused. It is however, the one that is used by other scripts (T09 in trello). I think we should remove this variable from here, and find a way to expose a better interface of `FindUnassignedApp` through this task.

Reply:
Removed!